lightning/src/ln/channelmanager.rs

```diff
@@ -3890,23 +3928,21 @@ where
 
 			claimable_amt_msat += htlc.value;
 		}
+		mem::drop(per_peer_state);
```

Comment:
is this still explicitly necessary considering we're no longer accessing per_peer_state afterwards in this function?

Reply:
never mind, I see it is due to the nesting